pdc_client/__init__.py

```diff
@@ -97,7 +98,7 @@ class PDCClient(object):
     connections. The authentication token is automatically retrieved (if
     needed).
     """
-    def __init__(self, server, token=None, develop=None, ssl_verify=None, page_size=None):
+    def __init__(self, server, token=None, develop=None, ssl_verify=None, page_size=None, page=None):
```

Comment:
Please remove the `page` parameter from constructor. It really doesn't make sense for all requests of a PDCClient instance to fetch only particular page.

Correct me if I'm wrong but the `page` attribute is only useful when iterating over results, I don't see any other real use case.

Reply:
Add new class here for the page option of pdc command.
Please help to review.